R/calcPotentialWindOn.R

```diff
@@ -1,68 +1,59 @@
 #' Calculate wind onshore potential
-#' 
+#'
 #' Provides wind onshore potential data
-#' 
-#' 
+#'
+#'
 #' @return wind onshore potential data and corresonding weights as a list of
 #' two MAgPIE objects
 #' @author Lavinia Baumstark
 #' @seealso \code{\link{calcOutput}}, \code{\link{readNREL}},
 #' \code{\link{convertNREL}}, \code{\link{readSource}}
 #' @examples
 #' 
-#' \dontrun{ 
+#' \dontrun{
 #' calcOutput("PotentialWindOn")
 #' 
 #' }
 calcPotentialWindOn <- function() {
```

Comment:
FYI, the output is exactly identical from what I can see. Description says "PotentialWind: renaming of variables".
Should sth have changed here?